### ISSUE TITLE
Favorite folders sometimes disappear

### ISSUE THREAD

**Issue by kerim** [sethyuan/logseq-plugin-another-embed#9]
The feature to show favorites as folders sometimes disappears and I have to disable/re-enable the plugin for them to show up again. It has happened to me enough times now that it seems to be a regular thing, so I thought I should report it.

**Comment by sethyuan** [comment]
Can you open the console and upload an screenshot of the errors you see next time it happens? Also, what version of Logseq do you use? Be sure to use the latest.

**Comment by kerim** [comment]
I'm always using the latest version. I don't see any errors related to this plugin. It is possible that some other plugin is causing a conflict I suppose, but I haven't had time to test that...

**Comment by kerim** [comment]
The only thing I notice is a shortcut conflict error on restart: ":plugin.logseq-another-embed/$shortcut$mod_9 meta+9 Keyboard shortcut conflicts with existing shortcut: plugin.logseq-another-embed/$shortcut$mod_9"

**Comment by sethyuan** [comment]
> The only thing I notice is a shortcut conflict error on restart: ":plugin.logseq-another-embed/$shortcut$mod_9 meta+9 Keyboard shortcut conflicts with existing shortcut: plugin.logseq-another-embed/$shortcut$mod_9"

This one is fine. It's unrelated.

So far, there is not enough information to guess what issue it could be. What were you doing when favorites stopped working?

**Comment by kerim** [comment]
If I can figure that out I will tell you!

**Comment by kerim** [comment]
All I know is it happens very frequently. But I only notice it when switching back to Logseq from another app, so I don't have a sense of having "done anything" that might cause it. (But this could be an illusion.)

**Comment by sethyuan** [comment]
I use this feature everyday too, but I'm not observing what you experienced.

**Comment by kerim** [comment]
So I noticed it again when I opened the app this morning. Wondering if the fact that I use emojis in the names of my "folder" pages has anything to do with it?

**Comment by sethyuan** [comment]
@kerim I created some pages with emoji in names and found no problems at all. Could it be a CSS conflict? Maybe some themes or plugins affected the left sidebar too?

**Comment by kerim** [comment]
Hmm. Possibly one of the Awesome plugins, they do a lot to the UI and might be causing a conflict. I'll try to test this...

**Comment by kerim** [comment]
I can confirm that the problem starts with each new day. I am seeing errors in the console related to the block calendar plugin, so I’m going to disable that for a bit to see if it helps, since it does seem date related...

**Comment by kerim** [comment]
So, for the first time, I didn't have the problem this morning. Two things changed from yesterday with regard to other plugins:

1. The WordCount plugin, which had been producing errors, was updated to fix those errors
2. I disabled the Awesome UI plugin, which also makes changes to the sidebar.

I suspect 2 was more important than 1, but thought I would mention both. In any case, it does seem like a conflict with other plugins, so I will close this issue.
